MeshRender/OpenGLTest/main.cpp

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <CommonMeshRepresentation.h>
+#include "Renderer.h"
+GLFWwindow* window;
+
+
+int main(void)
+{
+	Renderer* renderer = new Renderer();
+	MeshIS::Model::Common::CommonMeshRepresentation commonMesh;
+	Vertex vertex;
+	//przykladowe dane; szescian
+	for (int j = 0; j < 3; j++) {
+		for (int i = 0; i < 100; i++)
+		{
+			vertex[j] = (double)i / 20.;
```

Comment:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es49-if-you-must-use-a-cast-use-a-named-cast